x-pack/plugins/index_management/public/application/sections/home/template_list/template_details/tabs/tab_summary.tsx

```diff
@@ -153,7 +158,13 @@ export const TabSummary: React.FunctionComponent<Props> = ({ templateDetails })
               </EuiDescriptionListTitle>
               <EuiDescriptionListDescription>
                 {ilmPolicy && ilmPolicy.name ? (
-                  <EuiLink href={getILMPolicyPath(ilmPolicy.name)}>{ilmPolicy.name}</EuiLink>
+                  <EuiLink
```

Comment:
Just a note that we need an `href` here too, so users can copy the url or open the link in a new tab. If you grep for `reactRouterNavigate` you can see how we use this utility to provide both an href and an onClick.

Reply:
@cjcenizal thanks for pointing this out! From what I can see, `reactRouterNavigate` doesn't work when navigating to other apps. I came across https://github.com/elastic/kibana/issues/67697, which suggests using `history.parentHistory`. This works, however, it is private in `ScopedHistory`, so I get a TS error when I use it. I will change it back to a `href` for now, but there will be a full page refresh.